src/main/java/com/automation/school/ad_main/Collection/Address.java

```diff
@@ -0,0 +1,23 @@
+package com.automation.school.ad_main.Collection;
+
+public class Address {
+    public static String country;
+    public static String city;
+    public static String street;
+    //String data[] = {country, city, street};
+    //public String getFullAddress;
+
+    public static String getFullAddress()
+    {
+        String FullAddress = country +" " + city + " "+ street;
```

Comment:
First of all- codestyle, arrange it by pressing crtl+alt+L, 
Second - variable names always starts with lowercase 
And finally line 12 can be inline with line 13:
> return country + " " + city + " "+ street;

Reply:
Code was aliened. Method is modified
 public static String getFullAddress() {
        return country + " " + city + " " + street;
    }